### ISSUE TITLE
[Paladin] Judgement of The Crusader doesn't give Holy Damage Bonus

### ISSUE THREAD

**Issue by Tolgren** [ccshiro/cc-buglist#912]
It should give around 219 Holy Damage Bonus on highest Rank, still Seal of Blood does only a small amount more damage as without the Judgement.

---

Without Judgement:

![wowscrnshot_070515_185305](https://cloud.githubusercontent.com/assets/13187502/8512533/26b6a7fc-2347-11e5-92b8-0f4ad0c9c68c.jpg)

Seal of Blood does around 230-260 damage.

---

With Judgement:

![wowscrnshot_070515_185444](https://cloud.githubusercontent.com/assets/13187502/8512536/56818196-2347-11e5-96b1-d292718a77c0.jpg)

Seal of Blood does around 230-290 damage, basically the same.

**Comment by Ensipid** [comment]
Same problem happens with Seal of Command.
![ccsoc](https://cloud.githubusercontent.com/assets/13189833/8513574/820d702a-233f-11e5-9877-2aa49f03422a.png)

Problem doesn't occur with Seal of Righteousness.
![ccsor](https://cloud.githubusercontent.com/assets/13189833/8513579/a8aafd56-233f-11e5-9910-9604ad7ce5fb.png)

**Comment by ccshiro** [comment]
Tested Seal of Command and Seal of Righteousness, they have the intended scaling with Judgement of the Crusader up.

As for Seal of Blood, whether or not it should give more retaliation damage with Judgement of the Crusader up is not entirely clear. The seal damage with the judgement works a bit strange, and I could see it as possible that it shouldn't scale, too. Then again, I could see the opposite also being true. In summary, we should look for some evidence of this I think.

**Comment by Tolgren** [comment]
Regarding Seal of Blood:

Seal mechanic discussion on EJ in 2007: http://forums.elitistjerks.com/topic/9664-paladin-seal-mechanics/

It seems that Seal of Blood damage has 0% spell damage coefficient, which means it indeed doesn't benefit from the +Holydmg bonus from Judgement of the Crusader.

Also, I didn't test it Seal of Command, but as a side note, Seal of Command has a 20% spell damage coefficient, and 29% for _holy_ spell damage. source: https://web.archive.org/web/20060328090156/http://forums.worldofwarcraft.com/thread.aspx?fn=wow-paladin&t=647087&p=1&tmp=1#post647087

**Comment by Barroth** [comment]
Found this question on the wow forums:
"...3. How come JotC (additional holy damage) doesn't really give a damage boost to seal of blood? 

3) Seal of Blood itself has no spell damage coefficient ... "

http://web.archive.org/web/20080221133542/http://forums.worldofwarcraft.com/thread.html?topicId=104544938&sid=1&pageNo=6

Seal of Command already has 20% spelldamage and 29% holy scaling.

**Comment by CCjudgement** [comment]
I have solved this bug report in regards to seal of blood:

Seal of Blood SHOULD:
Be based on weapon damage/physical stats
Deal holy damage
Be able to crit
Crits should proc vengeance
Should NOT scale with spell damage

Judgement of Blood SHOULD:
Be affected by physical stats i.e melee crit/hit
SHOULD scale with spell damage at 43% (The same as Judgement of Command)

TL;DR: Judgement of crusader should not affect seal of blood, It should however affect judgement of blood at 43%.

1st Source
[http://web.archive.org/web/20080221133542/http://forums.worldofwarcraft.com/thread.html?topicId=104544938&sid=1&pageNo=6](url)

> Seal of Blood itself has no spell damage coefficient, but SoComm only has 20%. The Judgement (Seal of Blood) has a 43% spell damage coefficient, though, equivalent to Judgement of Command, meaning it will always have higher Judgement damage because of its higher base damage.
> 
> 800-1000 damage, speed 3.6, crit chance 30% (pre-patch) 
> 350 bonus spell damage 
> 
> ... Seal of Command range: 830-1,037 Holy damage. Avg: 933.5 
> ... Seal of Blood range: 414-519 Holy damage. Avg: 466.5 
> 
> ... Judgement of Command range: (228 to 252) + (0.43(350)) = 379-403 Holy damage. Avg: 391 
> ... Judgement of Blood range: (331 to 362) + (0.43(350)) = 482-513 Holy damage. Avg: 497 
> 
> White damage over 5 minutes: (Avg)_(total seconds / weapon speed) = (900)_(300 / 3.6) = 75,000 
> ... Critical white damage: (total damage)_(critical chance / 100) = (75k)_(0.3) = 22,500 
> ... ... Total white damage: 22,500 + 75,000 = 97,500 
> 
> Seal of Command damage: (Avg)_(ppm) = (933.5)_(7_5) = 32,673 
> ... Critical SoC damage: (32,673)_(0.3) = 9,802 
> ... ... Total SoC damage: 9,802 + 32,673 = 42,475 
> Seal of Blood damage: (Avg)_(total seconds / weapon speed) = (466.5)_(300 / 3.6) = 38,875 
> ... Critical SoB damage: (38,875)*(0.3) = 11,663 
> ... ... Total SoB damage: 11,663 + 38,875 = 50,538 
> 
> Judgement of Command damage: (Avg)_(300 / 8) = (391)_(37.5) = 14,623 
> ... Critical JoC damage: (14,623)_(0.3) = 4,399 
> ... ... Total JoC damage: 4,399 + 14,623 = 19,023 
> Judgement of Blood damage: (497)_(37.5) = 18,638 
> ... Critical JoB damage: (18,638)*(0.3) = 5,591 
> ... ... Total JoB damage: 5,591 + 18,638 = 24,229 
> 
> This doesn't take into account Vengeance bonus, Sanctity Aura bonus, or other miscellaneous bonuses. Plug in higher spell damage numbers and see what ya get.

2nd Source:
[http://www.engadget.com/2007/11/27/the-light-and-how-to-swing-it-command-vs-blood/](url)
27/11/07

3rd Source:
[http://wowwiki.wikia.com/wiki/Seal_of_Blood](url)

**Comment by ccnim** [comment]
So is it working as it should currently or does the judgement of blood not scale?
